src/EasyPdf.php

```diff
@@ -12,7 +12,7 @@ class EasyPdf
      *
      * @var TCPDF
      */
-    protected $pdf;
+    public $pdf;
```

Comment:
why this needs to be public?

Reply:
Because i want use all of tcpdf methods. This helper methods not have all tcpdf methods and i don't want add getter-setter for all tcpdf methods.